fs/localfs/local_fs.go

```diff
@@ -128,8 +128,8 @@ type entryWithError struct {
 	err   error
 }
 
-func toDirEntryOrNil(basename, prefix string) (fs.Entry, error) {
-	fi, err := os.Lstat(prefix + basename)
+func toDirEntryOrNil(dirEntry os.DirEntry, prefix string) (fs.Entry, error) {
+	fi, err := os.Lstat(prefix + dirEntry.Name())
```

Comment:
> So as I was saying - with this change Windows will be failing 100% - there's some subtle difference between os.Lstat() and os.ReadDir() result, (which could be insignificant but the test is failing nonetheless).

@jkowalski The Windows integration test should pass now. I have tested this on my fork repository Actions workflow.

I tried changing `os.Lstat()` into `os.Stat()` and the Windows integration test passed as well. There is definitely something difference between `dirEntry.Info()` and `os.Stat() / os.Lstat()`